scopes/lanes/ui/dropdown/lanes-list.dropdown.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { HTMLAttributes, useRef, useEffect } from 'react';
+import classnames from 'classnames';
+import { Dropdown } from '@teambit/design.inputs.dropdown';
+import { LanesIcon } from '@teambit/lanes.ui.icon';
+import { useLanes } from '@teambit/lanes.hooks.use-lanes';
+import { DEFAULT_LANE, LanesModel } from '@teambit/lanes.ui.models';
+import { Icon } from '@teambit/evangelist.elements.icon';
+import { MenuLinkItem } from '@teambit/design.ui.surfaces.menu.link-item';
+
+import styles from './lanes-list.dropdown.module.scss';
+
+export type LanesListDropdownProps = {} & HTMLAttributes<HTMLDivElement>;
+
+export function LanesListDropdown({ className, ...rest }: LanesListDropdownProps) {
```

Comment:
this logic is indeed unrelated to lane selector. let's create a simple ui component for lane selector and use this from this main component. Let's call this one: teambit.lanes/navigation/lane-switcher